tests/cmd/stability/main.go

```diff
@@ -154,10 +154,16 @@ func main() {
 	oa.CheckTidbClusterStatusOrDie(cluster1)
 	oa.CheckTidbClusterStatusOrDie(cluster2)
 
-	//go func() {
-	//	oa.BeginInsertDataTo(cluster1)
-	//	oa.BeginInsertDataTo(cluster2)
-	//}()
+	defer func() {
```

Comment:
why not split two defer function?

Reply:
What's the point of split two defer functions?

Reply:
I get it

Reply:
just avoid to executing useless defer，in the other hand the errors in defer can be confused with real errors